.github/workflows/main.yml

```
@@ -31,25 +37,14 @@ jobs:
         username: vcr-bot
         password: ${{ secrets.GHCR_TOKEN }}
 
-    - name: Set up Docker Buildx
-      id: buildx
-      uses: docker/setup-buildx-action@v1
-
-    - name: Build and push
-      id: docker_build
-      uses: docker/build-push-action@v2
+    - name: Run GoReleaser
+      uses: goreleaser/goreleaser-action@v2
       with:
-        context: ./
-        file: ./docker/vhs/Dockerfile
-        platforms: linux/amd64
-        builder: ${{ steps.buildx.outputs.name }}
-        push: true
-        target: application
-        tags: |
-          ghcr.io/${{ github.repository }}:${{ github.sha }}
-          ghcr.io/${{ github.repository }}:edge
-        cache-from: type=local,src=/tmp/.buildx-cache
-        cache-to: type=local,dest=/tmp/.buildx-cache
+        version: latest
+        # TODO figure out appropriate args here
+        args: release --rm-dist
```

```suggestion
        args: release
```

`--rm-dist` should not be required when you run on github actions, since each job has it's VM / workspace.